source/Font.h

```diff
@@ -71,6 +71,7 @@ class Font {
 	GLint aspectI;
 	GLint positionI;
 	
+	int width;
```

Comment:
Forgot to initialize `width` in the member initializer list of the default constructor `Font::Font()` in `Font.cpp`.

Why use a constructor's member initializer list (in `Font::Font() :` define `width(0),`) instead of default member initialization (in the `class Font {` declare `int width = 0;`) and defaulting the default constructor (`Font::Font() = default;`)?
The latter is less error prone [and can be optimized in the STL].

Reply:
Thanks, fixed. I'll leave the initialization strategy for another time, though.